examples/CocoDataPyTorch.py

```diff
@@ -11,7 +11,7 @@ def image_to_byte_array(image: Image) -> bytes:
     return imgByteArr
 
 
-class CocoDataPytorch(PyTorchData):
+class CocoDataPyTorch(PyTorchData):
```

Comment:
This file and class are mentioned elsewhere.  Should those be fixed?

Reply:
OK I 'll do a find.